Fix / Connect Payment page copy plus styling

styrelius opened the pull request:
## What?

- New copy on `/connect-payment` page that hopefully makes it more clear that you are now a Hedvig member, but you still need to connect payment and download the app.
- Some styling fixes to adjust to the new text (many of the sentences are longer now), and to make it look better in more viewports.


## Why?

We've gotten feedback from the IEX team that it's not clear to people after they've signed that they actually are Hedvig members. These copy changes aim to address that issue.


_Referenced ticket(s): [GRW-276]_
<!-- If there is a Jira issue, add the id between brackets, and a link to that issue will be created. -->


<!-- Finally, if that makes sense, add screenshots and/or screen recordings below, preferably with headlines and/or descriptions -->

## Screenshots

**/se-en:**
<img width="1187" alt="2021-05-18-connect-payment-mobile-se-en" src="https://user-images.githubusercontent.com/42962286/118674986-8706aa00-b7fa-11eb-865c-c87360e768ee.png">

**/no:**
<img width="1653" alt="2021-05-18-connect-payment-mobile-no" src="https://user-images.githubusercontent.com/42962286/118675257-bcab9300-b7fa-11eb-95fe-96bdabac61e2.png">

**/se:**
<img width="1742" alt="2021-05-18-connect-payment-desktop-se" src="https://user-images.githubusercontent.com/42962286/118674090-d26c8880-b7f9-11eb-90b4-b2b78758e845.png">

**/no:**
<img width="1776" alt="2021-05-18-connect-payment-desktop-no" src="https://user-images.githubusercontent.com/42962286/118675284-c46b3780-b7fa-11eb-8532-754261ff590d.png">


**/dk:**
<img width="1776" alt="2021-05-18-connect-payment-desktop-dk" src="https://user-images.githubusercontent.com/42962286/118674255-f16b1a80-b7f9-11eb-8903-a7393a4e93a3.png">

**/dk-en**
<img width="1776" alt="2021-05-18-connect-payment-desktop-dk-en" src="https://user-images.githubusercontent.com/42962286/118674292-f5973800-b7f9-11eb-80d0-46ed2c0ef14f.png">






[GRW-276]: https://hedvig.atlassian.net/browse/GRW-276

styrelius committed:
Download new translations/copy for the /conect-payment page, and change two text keys in ConnectPayment.tsx

styrelius committed:
Fix a bunch of styling/layout issues in ConnectPayment.tsx and AdyenCheckout.tsx

styrelius committed:
Download new 'translations' for the Swedish connect payment CTA button, in both Swedish and English (being clear that it's direct debit, not just any payment method that you will connect)

styrelius committed:
Fix styling of TrustlyCheckout CTA button

styrelius committed:
Some more styling fixes after talking to Mange

robinandeer reviewed (approved):
Nice work! A lot of changes to try to keep track of but I can't really find something that needs changing.

I like that you are cleaning up the paddings to be more like "8rem" rather than "7.25rem"!

styrelius commented:
> Nice work! A lot of changes to try to keep track of but I can't really find something that needs changing.
> 
> I like that you are cleaning up the paddings to be more like "8rem" rather than "7.25rem"!

Haha I know, I'm sorry - I moved too much things around 😳 

Regarding the `rem` thing - I think we should scrap that altogether since it makes no sense to use it like we do it. IMO we should just have fixed pixel values for all measurements centralized. But since that requires more of a "helhetsgrepp" and it hasn't been popular before when I've replaced `rem` with `px` I tried to make the best of it 😅
